styles/styles.css

```
@@ -1,7 +1,284 @@
 @import url("./colors.css");
 @import url("./fonts.css");
 
-.test {
-  color: var(--color-primary);
+/* ========================== HOME PAGE ================= */
+
+
+
+/* ==== MOBILE ==== */
+
+@media (max-width: 567px) {
+  .grid-container {
+    grid-template-columns: 1fr;
+  }
+
+  .poppins-style {
+    text-align: center;
+  }
+
+  .banner-style {
+    height: auto;
+    padding: 20px;
+  }
+
+  .home-page .image-container img {
+    width: 170px;
+    height: 150px;
+  }
+  
+  .home-page .text-img-container {
+    justify-content: center;
+    width: fit-content;
+    padding: 10px;
+  }
+
+  #title-banner {
+    font-size: 20px;
+    margin: 0 0 5px 0;
+  }
+
+  .icons-style {
+    gap: 10px;
+    margin: 0;
+  }
+  
+  .custom-icon {
+    width: 15px;
+    height: 15px;
+  }
+  
+  .title-products{
+    font-size: 30px;
+  }
+
+  .banner-style .poppins-style {
+    font-size: 12px;
+  }
+}
+
+/* ==== DESKTOP ==== */
+body {
+  /* ==== COMMON ==== */
+  font-family: var(--font-primary);
+  padding: 30px;
+  margin: 0;
+}
+
+/* ==== ADS (FIRST SECTION) ==== */
+.home-page .img-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.home-page .img-container img {
+  margin: 10px;
+  max-width: 100%;
+  border-radius: 20px;
+  /* ==== COMMON ==== */
+  box-shadow: 0 4px 8px var(--color-grey);
+}
```

(Solo para la segunda imagen), creo que un shado mas sueve un borde clarito le quedaría melo 


```suggestion
.home-page .img-container img {
  margin: 10px;
  max-width: 100%;
  border-radius: 20px;
  /* ==== COMMON ==== */
  box-shadow: 0 4px 8px var(--color-grey);
}

.home-page .img-container img:nth-of-type(2) {
  box-shadow: 1px 4px 29px -23px rgba(0, 0, 0, 0.75);
  border: 2px solid #d8d2be;
}
```